### PR TITLE
ETC.tsv Lines 17244-17340

### PR DESCRIPTION
**Pull request by ttgmichael**
This should cover most of the blank lines from November.

Let me know if I messed anything up.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -17234,115 +17234,115 @@ ETC_20151102_017235	The entered name is not a Nexon.com member.
...
-ETC_20151102_017269
-ETC_20151102_017270
-ETC_20151102_017271
+ETC_20151102_017254	These traces have already been examined 
+ETC_20151102_017255	You examined the burnt message{nl}it appears it caught fire some number of days ago
+ETC_20151102_017256	You examined the extinguished flame{nl}Looks like someone was in a hurry as the traces of the fire remain
+ETC_20151102_017257	You examined the discarded bag{nl}It appears the person took what they needed and left
+ETC_20151102_017258	Released the ominous magic circle
+ETC_20151102_017259	Obtained the Chasers' Pouch
+ETC_20151102_017260	Sleep
```
Comment (JayShee) [inline ETC.tsv]: I believe this is "Jolly"

Comment (ttgmichael) [inline ETC.tsv]: Ah, my bad. Good catch! :p

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**